Update the repo?

paetling opened the pull request:
@phunt I stumbled upon this repo last week and decided I wanted to use it to help monitor our zookeeper cluster.  I forked the repo and did a bunch of cleanup:
1. Got rid of compiled binary file and switched to using [kazoo](https://github.com/python-zk/kazoo)
2. Upgraded to use django 1.9
3. Added a requirements.txt file
4. Made it really easy to run [gunicorn](http://gunicorn.org/) in front of the server.  

I have some debug statements I need to take out, remove the Dockerfile, and add back in some files, but I was wondering if you were interested in merging these updates back into this repo.

paetling committed:
make it work using zookeeper and across the interwebs

paetling committed:
allow for ZOOKEEPER_SERVERS environment variable

paetling committed:
add in Dockerfile so we can build this image

paetling committed:
have last command run be start of the server

paetling committed:
runserver better

paetling committed:
add gunicorn and get rid of binary file

paetling committed:
remove gitignore

paetling committed:
try and refactor to new django

paetling committed:
remove dumb file

paetling committed:
make it work

paetling committed:
change dockerfile to include application

paetling committed:
make it work

paetling committed:
remove static

paetling committed:
allow all hosts, no debug

paetling committed:
make it ipv6 compatible

phunt commented:
Hi @paetling . Sometime back the upstream Apache project (ZooKeeper) took on continued development of the dashboard. See https://issues.apache.org/jira/browse/ZOOKEEPER-701 (notice it kinda morphed into something else...)

As a result I haven't been actively developing this project, however I'm happy to pull changes if people are still finding it interesting.

I notice immediately that you've removed some important files. The license, readme, etc... I would not feel comfortable merging that change, e.g. removing Apache licensing from the project. Would you be able to review/address and submit a squashed patch?

I notice docker support was added, which is quite cool. Adding details on that to the README would be nice. :-)

Regards!

paetling committed:
do not crash if leader says it is not reporting stats at the moment

paetling committed:
add another requirement

magmax committed:
Python 3 compatibility

magmax committed:
Fix StringIO init

Commit:
Merge pull request #1 from magmax/master

Python 3 compatibility

magmax committed:
More on python 3 compatibility

magmax committed:
Fix the list of servers, remove prints and add tracebacks

magmax committed:
small fix

magmax committed:
Improve style and navigation

Added bootstrap dependency
Created a base class to allow a common style for everything.
Improved navigation with breadcrumbs
Tables to definition lists for properties
Information links as buttons
Other visual improvements

Commit:
Merge pull request #2 from magmax/master

More on python 3 compatibility

Commit:
Merge pull request #3 from magmax/master

Improve style and navigation

thieman committed:
Add back original license

daymade committed:
Fix: Fix kazoo bug on windows platform when viewing znode tree

* issue: kazoo: error 10038 An operation was attempted on something that is not a socket
* please see  https://groups.google.com/d/msg/python-zk/EHdF-c_N0OI/6VTWuTHhdnUJ

Commit:
Merge pull request #4 from tsongknows/master

Fix: Fix kazoo bug on windows platform when viewing znode tree
